packages/tabs/src/Tabs.ts

```diff
@@ -16,7 +16,7 @@ import {
     CSSResultArray,
     TemplateResult,
     PropertyValues,
-} from 'lit-element';
+} from '@spectrum-web-components/base';
 import { Tab } from './Tab.js';
 import { Focusable, getActiveElement } from '@spectrum-web-components/shared';
 
```

Comment:
I also noticed that the `seletionIndicator` line was not showing up for the `rtl` tabs. 

We may need to go into `updateSelectionIndicator()` and add this conditional to cover the rtl case:

```
if (this.direction === 'horizontal' && this.dir === 'rtl') {
    const width = tabBoundingClientRect.width;
    const offset = tabBoundingClientRect.right - parentBoundingClientRect.right - width;

    this.selectionIndicatorStyle = `transform: translateX(${offset}px) scaleX(${width});`;
} 
```

Reply:
Yeah, rather than have two versions I overrode the RTL CSS so that the same positioning would work on the indicator in both contexts.